cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/tekton-dashboard/clusterrolebinding.yaml

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
Can you break these up as per the format described here: https://github.com/operate-first/apps/tree/master/cluster-scope#application-folder-structure

Same with the clusterroles. 


(seems tedious but this is the current structure, we're debating the cons of this format [here](https://github.com/operate-first/SRE/issues/369) if you're interested )

Reply:
ack, I have updated with more breakdowns now
does it fit right ?